addons/mail/views/discuss_channel_views.xml

```diff
@@ -8,35 +8,25 @@
             <field name="priority" eval="10"/>
             <field name="arch" type="xml">
                 <kanban>
-                    <field name="id"/>
-                    <field name="description"/>
                     <field name="is_member"/>
                     <field name="group_ids"/>
                     <field name="active"/>
                     <templates>
                         <t t-name="kanban-description">
```

Comment:
this is dead code?? Where is this template called?

Reply:
I believe it is indeed dead code now. I was also confused about this, as I couldn't find its implementation anywhere else in Odoo.